Dockerfile

```diff
@@ -5,55 +5,30 @@ ENV LANG en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 
 # Stuff for building steem-python
-ARG BUILD_ROOT=/buildroot
-ARG BUILD_OUTPUT=${BUILD_ROOT}/build
+ARG BUILD_ROOT=/build
```

Comment:
obviously you're just renaming a thing here so this doesn't hold up the PR, but why are we using build args to begin with?

Reply:
i found it like that; it could probably be removed. docker is only used presently for running the tests.